tests/Unit/Evolution/DiscontinuousGalerkin/SlopeLimiters/Test_WenoHelpers.cpp

```diff
@@ -82,11 +97,215 @@ void test_oscillation_indicator_3d() noexcept {
   CHECK(indicator == approx(expected));
 }
 
+struct ScalarTag : db::SimpleTag {
+  using type = Scalar<DataVector>;
+  static std::string name() noexcept { return "Scalar"; }
+};
+
+template <size_t VolumeDim>
+struct VectorTag : db::SimpleTag {
+  using type = tnsr::I<DataVector, VolumeDim>;
+  static std::string name() noexcept { return "Vector"; }
+};
+
+void test_reconstruction_1d() noexcept {
+  const double neighbor_linear_weight = 0.001;
+  const auto mesh = Mesh<1>({{5}}, Spectral::Basis::Legendre,
```

Comment:
make `5` a variable instead of a magic number used in several places (`constexpr size_t`)

Reply:
I don't like making this a variable, because that suggests (imo) that changing its value is a good idea, whereas here the entire test function would need to be adapted for a different resolution.